### PR TITLE
tutorial: toxicity classifier

### PR DESCRIPTION
**Pull request by jarulraj**
- [x] Added a formatting function check (in formatter.py) for notebooks (checks Colab links, empty cells, etc.)

### CODE REVIEW

**Comment by jarulraj** [comment]
This PR is ready for review

**Comment by jarulraj** [comment]
#709 needs to be resolved before this PR can be merged. This PR adds two tutorials.

File: eva/udfs/toxicity_classifier.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+from transformers import (
+    AutoModelForSequenceClassification,
+    AutoTokenizer,
+    TextClassificationPipeline,
+)
+
+from eva.udfs.abstract.abstract_udf import AbstractClassifierUDF
+
+
+class ToxicityClassifier(AbstractClassifierUDF):
```
Comment (gaurav274) [inline eva/udfs/toxicity_classifier.py:27]: We shouldn't need this file with the current HF design

**Comment by gaurav274** [comment]
We can convert all `print(response)` to `response.as_df()`

File: script/formatting/pre-push.sh [hunk 2]
```diff
@@ -14,11 +14,11 @@ if [ $exit_status -ne 0 ]; then
...
 fi
 
-if ! git diff-index --quiet HEAD --; then
-    echo "Code was reformatted or you have unstaged changes." 
-    echo "Please verify and stage the changes."
-    echo "List of files updated."
-    git --no-pager diff --name-only
-    exit 1
-fi
+# if ! git diff-index --quiet HEAD --; then
```
[the remark this answers is not in the document]
Comment (jarulraj) [reply script/formatting/pre-push.sh]: Resolved.

**Pull request merged**